autogpt_platform/frontend/src/lib/autogpt-server-api/baseClient.ts

```diff
@@ -70,8 +70,16 @@ export default class BaseAutoGPTServerAPI {
     return this._get("/templates");
   }
 
-  getGraph(id: string, version?: number): Promise<Graph> {
-    const query = version !== undefined ? `?version=${version}` : "";
+  getGraph(
+    id: string,
+    version?: number,
+    hide_secrets?: boolean,
+  ): Promise<Graph> {
+    let query = version !== undefined ? `?version=${version}` : "";
+    if (hide_secrets !== undefined) {
+      query += query ? "&" : "?";
+      query += `hide_secrets=${hide_secrets}`;
+    }
     return this._get(`/graphs/${id}` + query);
```

Comment:
nit: `_get` has a `query: Record<string, any>` parameter:
```ts
  private async _get(path: string, query?: Record<string, any>) {
    return this._request("GET", path, query);
  }
```
so you can simply do:
```suggestion
    let query: Record<string, any> = {};
    if (version !== undefined) {
      query["version"] = version;
    }
    if (hide_secrets !== undefined) {
      query["hide_secrets"] = hide_secrets;
    }
    return this._get(`/graphs/${id}`, query);
```